### DISCUSSION TITLE
Problem: email recipient can't see sender's avatar

### DISCUSSION THREAD

**Discussion by rhonnysharma** [clearpool-finance/community#15] [Ideas]
Bug Description:
The Bimi record is missing from the email header, resulting in the failure to display a brand logo in the recipient's email client. This bug affects the visual representation of emails and compromises brand authenticity and identification.

Steps to Reproduce:

1) Navigate this url https://dmarcly.com/tools/bimi-record-checker 
and enter your domain name clearpool.finance

Expected Result:
The recipient's email client should display the brand's logo next to the sender's name or email address, providing visual authentication and identification.

Actual Result:
The recipient's email client does not display the brand's logo. Instead, a generic placeholder or no image is shown next to the sender's name or email address.

Impact:
The absence of the brand's logo due to the missing Bimi record affects the visual representation of emails, potentially leading to reduced brand recognition and user trust. It also undermines the purpose of Bimi, which is to provide visual authentication and identification for email recipients.

**Comment by georgeciubotaru** [comment]
Hi, @rhonnysharma Thank you for your report. I will pass the issue to our development team and let you know the resolution.
@VarinderClearpool FYI
